docs/tables/aws_cost_by_tags.md

```diff
@@ -0,0 +1,66 @@
+# Table: aws_cost_by_tags
+
+Amazon Cost Explorer helps you visualize, understand, and manage your AWS costs and usage. The `aws_cost_by_tags` table provides a simplified view of cost by tags in your account. ou must specify a granularity (`MONTHLY`, `DAILY`) and `tag_key_1`. `tag_key_2` is optional for the table.
+
+Note that [pricing for the Cost Explorer API](https://aws.amazon.com/aws-cost-management/pricing/) is per API request - Each request will incur a cost of $0.01.
+
+## Examples
+
+### Basic info
+
+```sql
+select
+  tag_key_1,
+  tag_value_1,
+  period_start,
+  blended_cost_amount::numeric::money,
+  unblended_cost_amount::numeric::money,
+  amortized_cost_amount::numeric::money,
+  net_unblended_cost_amount::numeric::money,
+  net_amortized_cost_amount::numeric::money
+from
+  aws_cost_by_tags
+where
+  granularity = 'DAILY'
+and
+  tag_key_1 = 'Name';
+```
+
+### Min, Max, and average daily unblended_cost_amount by tag
+
+```sql
+select
```

Comment:
Im getting EOF errors (`Error: rpc error: code = Unavailable desc = error reading from server: EOF (SQLSTATE HV000)` running this with an aggregator...

Reply:
working for me now